data/enemy/enemies.py

```diff
@@ -0,0 +1,193 @@
+# ruff: noqa
+from typing import Self
+from data.enemy.enemies import *
+
+
+class Enemies:
+    MAPPINGS = {
+        "f4032b2323bc31d4590cf5197db3c3f1": Abstarak(),
```

Comment:
i'm assuming this may instantiate every enemy when we call Enemies().get

Is it possible in python to reference the object but not init it? then init it on the get?


We will need a new enemy object on every request - the memory code will check unique ids every frame, and if one doesn't exist it will use the guid to new an object -
it will also instead remove objects where the uid no longer exists.